kategory-core/src/test/kotlin/kategory/data/OptionTest.kt

```diff
@@ -60,6 +60,31 @@ class OptionTest : UnitSpec() {
             Option.fromNullable(a) shouldBe None
         }
 
+        "toOption should work for both null and non-null values of nullable types" {
+            forAll { a: Int? ->
+                // This seems to be generating only non-null values, so it is complemented by the next test
```

Comment:
You can combine generators using `Gen.oneOf()`. Try combining Gen.int() and create one for null.